components/StyledInput.js

```diff
@@ -37,14 +37,21 @@ const StyledInput = styled.input`
   border-style: ${props => (props.bare ? 'none' : 'solid')};
   box-sizing: border-box;
   outline: none;
+  background-color: ${themeGet('colors.white.transparent.72')};
 
   &:disabled {
     background-color: ${themeGet('colors.black.50')};
     cursor: not-allowed;
   }
 
-  &:focus, &:hover:not(:disabled) {
+  &:focus{
     border-color: ${themeGet('colors.primary.300')};
+    background-color: white;
```

Comment:
What is the goal you're trying to achieve with this line?

Reply:
Earlier I was using a lighter black(gray) for both the prepend currency and the input. But I changed it to white from the last feedback I got.

Reply:
I think it's white per default. If there's no visual changes in this block then we should revert to the default `&:focus, &:hover:not(:disabled)`